### PR TITLE
Use util to handle versions and update doc links

### PR DESCRIPTION
**Pull request by AileenCGN**
refs #116

- Added a util with a version file, that can take properties and handle `latest` versions and other versions in a more gneric way
- Added functionality that replaces old document links with the new ones, based on the `latest` version number.

### CODE REVIEW

File: lib/specs/latest.js [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
 
-rules = {};
-
-rules = _.merge(previousRules, rules);
+// Merge the previous rules into the new rules, but overwrite any specified property,
+// as well as adding any new rule to the spec.
+// Furthermore, replace the usage of the old doc URLs that we're linking to, with the
+// new version.
+rules = _.each(_.merge(previousRules, rules), function replaceDocsUrl(value) {
+    value.details = value.details.replace(prevDocsBaseUrlRegEx, latestDocsBaseUrl);
```
Comment (kirrg001) [inline lib/specs/latest.js]: There is a "g" missing the regex. I can still see some 1.25 links when i scan the latest version.

File: lib/specs/latest.js [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
 templates = _.union(previousTemplates, templates);
 
-rules = {};
-
-rules = _.merge(previousRules, rules);
+// Merge the previous rules into the new rules, but overwrite any specified property,
+// as well as adding any new rule to the spec.
+// Furthermore, replace the usage of the old doc URLs that we're linking to, with the
+// new version.
+rules = _.each(_.merge(previousRules, rules), function replaceDocsUrl(value) {
```
Comment (kirrg001) [inline lib/specs/latest.js]: When i scan with 1.0, i am not seeing 1.25 links in the report?

Comment (AileenCGN) [reply lib/specs/latest.js]: Hmm... I do...

Comment (AileenCGN) [reply lib/specs/latest.js]: No. It seems to work fine for me. I just logged the wrong rules. Can you show me an example?

File: lib/specs/latest.js (hunk 1 above)
Comment (AileenCGN) [reply lib/specs/latest.js]: Thanks! Added and pushed. It should be correct now.

File: lib/specs/latest.js (hunk 2 above)
Comment (kirrg001) [reply lib/specs/latest.js]: I pulled again and now it works. No idea 🤷🏼‍♂️

**Pull request merged**